lib/matplotlib/tests/test_ticker.py

```diff
@@ -613,3 +613,176 @@ def test_latex(self, is_latex, usetex, expected):
         fmt = mticker.PercentFormatter(symbol='\\{t}%', is_latex=is_latex)
         with matplotlib.rc_context(rc={'text.usetex': usetex}):
             assert fmt.format_pct(50, 100) == expected
+
+
+class TestTransformFormatter(object):
+    def transform1(self, x):
+        return -x
+
+    def transform2(self, x):
+        return 2 * x
+
+    @pytest.fixture()
+    def empty_fmt(self):
+        return mticker.TransformFormatter(self.transform1)
+
+    @pytest.fixture()
+    def fmt(self):
+        fmt = self.empty_fmt()
```

Comment:
This could take `empty_fmt` as an argument to make it clear it uses the fixture as well.